server.js

```diff
@@ -54,12 +54,14 @@ app.use('/', auth, cors(), morgan('combined'), graphqlHTTP(request => {
   loaders.clearAll();
 
   const accessToken = request.headers['x-access-token'];
+  const userID = request.headers['x-user-id'];
```

Comment:
I guess so, I’m not familiar with the authentication setup in Force, but @broskoski [seemed to know](https://github.com/artsy/gravity/issues/10229#issuecomment-232673536).